src/Module/Admin/Summary.php

```diff
@@ -85,20 +86,27 @@ public static function content()
 			try {
 				$stream = $fileSystem->createStream($file);
 
-				if (is_file($stream) &&
-				    !is_writeable($stream)) {
-					$warningtext[] = L10n::t('The logfile \'%s\' is not writable. No logging possible', $stream);
+				if (!isset($stream)) {
+					throw new InternalServerErrorException('Stream is null.');
 				}
 
 			} catch (\Throwable $exception) {
 				$warningtext[] = L10n::t('The logfile \'%s\' is not usable. No logging possible (error: \'%s\')', $file, $exception->getMessage());
 			}
 
-			$stream = Config::get('system', 'dlogfile');
+			$file = Config::get('system', 'dlogfile');
 
-			if (is_file($stream) &&
-			    !is_writeable($stream)) {
-				$warningtext[] = L10n::t('The logfile \'%s\' is not writable. No logging possible', $stream);
+			try {
+				if (!empty($file)) {
+					$stream = $fileSystem->createStream($file);
+
+					if (!isset($stream)) {
+						throw new InternalServerErrorException('Stream is null.');
+					}
+				}
+
+			} catch (\Throwable $exception) {
+				$warningtext[] = L10n::t('The developer logfile \'%s\' is not usable. No logging possible (error: \'%s\')', $file, $exception->getMessage());
```

Comment:
I don't think that "logging" is a *developer* tool. I would just remove that word from the warning.

Reply:
Why not?

Reply:
It's not the "default" logfile, which is possible to set in the admin-panel and its called `dlogfile`, so I thought d stands for developer :-)

how about "non-default" or "custom" or "technical" logfile?

Reply:
It stand for "debug" if I remember correctly.